.circleci/config.yml

```diff
@@ -19,22 +31,22 @@ shared: &shared
         - run:
             name: Tests
             command: npm test
-        - run:
-           name: Report Coverage
-           command: npm run coverage
+        - coverage
 
 jobs:
     node-latest:
+        docker:
+            - image: circleci/node:latest
         <<: *shared
 
-    node-10:
+    node-11:
         docker:
-            - image: circleci/node:10
+            - image: circleci/node:11
         <<: *shared
 
-    node-9:
```

Comment:
No need to worry about 9 as its not a LTS. Let's focus on 8, 10 (two recent LTS) and 11 (should help us notice issues that might apply to 12).